apps/condo/domains/meter/utils/taskSchema/importerResolver.js

```diff
@@ -74,14 +75,15 @@ function getColumnNames (format, locale) {
         { name: Value2ColumnMessage, type: 'string', required: false },
         { name: Value3ColumnMessage, type: 'string', required: false },
         { name: Value4ColumnMessage, type: 'string', required: false },
-        { name: ReadingSubmissionDateMessage, type: 'custom', required: true },
+        { name: ReadingSubmissionDateMessage, type: 'date', required: true },
         { name: VerificationDateMessage, type: 'date', required: false },
         { name: NextVerificationDateMessage, type: 'date', required: false },
         { name: InstallationDateMessage, type: 'date', required: false },
         { name: CommissioningDateMessage, type: 'date', required: false },
         { name: SealingDateMessage, type: 'date', required: false },
         { name: ControlReadingsDate, type: 'date', required: false },
         { name: PlaceColumnMessage, type: 'string', required: false },
+        { name: AutomaticColumnMessage, type: 'string', required: false },
```

Comment:
Should it be bool type instead?

Reply:
I'd like to abstract from any excel types. Empty cell means false, filled cell means true.

Reply:
Hm, as I see this settings not used since processing was moved to backend.

Reply:
Removed